book/svm.qmd

```diff
@@ -8,234 +8,248 @@ abstract: TODO (150-200 WORDS)
-
-#### SSVM-Rank {.unnumbered .unlisted} {#mod-svmem}
+where $CP$ is the set of comparable pairs defined by $CP = \{(i, j) : t_i < t_j \wedge \delta_i = 1\}$.
+Given the number of pairs, the optimization problem quickly becomes difficult to solve with a very long runtime.
+To solve this problem @VanBelle2011b found an efficient reduction that sorts observations in order of outcome time and then compares each data point $i$ with the observation with the next smallest *survival* time, skipping over censored observations.
+For example, consider the observed outcomes $\{(1, 1), (4, 0), (2, 1), (8, 0)\}$, sorting by outcome time gives  $\{(1, 1), (2, 0), (4, 1), (8, 0)\}$.
+The comparable pairs are then $(i=2, j=1)$, $(i=3,j=1)$, $(i=4,j=3)$ where the censored second outcome is skipped over when finding a pair for observation $3$.
+In this algorithm, the first observation is always assumed to be uncensored even if it was censored in reality.
+Using this reduction, the algorithm becomes
 
-The regression components of SSVM-Hybrid (\ref{eq:surv_ssvmvb2}) have been fully examined, now turning to the ranking components and setting $\mu = 0$. In this case the model reduces to
-
-**SSVM-Rank**\label{mod-ranksvmc}\label{mod:ssvmvb1}
 $$
 \begin{aligned}
-& \min_{\beta, \beta_0, \xi} \frac{1}{2}\|\beta\|^2 + C\sum_{i =1}^n \xi_i \\
+& \min_{\beta, \beta_0, \xi} \frac{1}{2}\|\beta\|^2 + \gamma\sum_{i =1}^n \xi_i \\
 & \textrm{subject to}
 \begin{dcases}
-& g(X_i) - g(X_{j(i)}) \geq T_i - T_{j(i)} - \xi_i, \\
-& \xi_i \geq 0, \quad \forall i = 1,...,n \\
+g(\xx_i) - g(\xx_{j(i)}) & \geq t_i - t_{j(i)} - \xi_i \\
+\xi_i & \geq 0 \\
+\forall i = 1,...,n
 \end{dcases}
 \end{aligned}
 $$
 
-with predictions
+where $j(i)$ is the observation with the next smallest survival time compared to $i$.
+Note the updated right hand side of the constraint, which plays a similar role to the $\epsilon$ parameter by allowing 'mistakes' in predictions without penalty.
+
+Prediction for a new observation $\xx^*$ are calculated as,
 
 $$
-\hatg(X^*) = \sum^n_{i=1} \alpha_i(K(X_i, X^*) - K(X_{j(i)}, X^*))
+\hatg(\xx^*) = \sum^n_{i=1} \alpha_i(K(\xx_i, \xx^*) - K(\XX_{j(i)}, \xx^*)) + \beta_0
```

Comment:
you switch from lower case x_{j(i)} to upper case X_{j(i)} here and I don't quiet follow

Reply:
just a typo